go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/Debian/dh-make-golang
 
-go 1.13
+go 1.16
 
 require (
+	github.com/charmbracelet/glamour v0.3.0
```

Comment:
Not already in Debian as I can see, we will need to package it

Reply:
Yay!  The Debian FTP masters approved the package, and it is now in unstable, and hopefully in testing soon: https://tracker.debian.org/pkg/golang-github-charmbracelet-glamour